web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/PublishedPagesSubscriber.php

```diff
@@ -182,8 +201,21 @@ public function modifyHtml(ModifyHtmlEvent $event) {
       // nodes, but that could negatively impact export performance.
       if ($decoded['Page ID'] && $hierarchy > 5) {
         $nid = $decoded['Page ID'];
-        $nodeEntity = \Drupal::entityTypeManager()->getStorage('node')->load($nid);
-        $url = $nodeEntity->toUrl()->toString();
+        if (!empty($nid)) {
+          if (substr($nid, 0, 2) !== 't_') {
```

Comment:
I believe this conditional is backward (should be `===` instead of `!==`)

Reply:
Oops! Your absolutely right! This is now fixed on this branch.